selftests/unit/test_utils_process.py

```diff
@@ -447,6 +447,64 @@ def test_process_get_owner_id_with_pid_not_found(self, stat_mock):
         self.assertIsNone(returned_owner_id)
         stat_mock.assert_called_with('/proc/%d/' % process_id)
 
+    @mock.patch('avocado.utils.process.safe_kill')
+    @mock.patch('avocado.utils.process.get_children_pids')
+    def test_kill_process_tree_nowait(self, get_children_pids, safe_kill):
+        safe_kill.return_value = True
+        get_children_pids.return_value = []
+        process.kill_process_tree(1)
+
+    @mock.patch('avocado.utils.process.safe_kill')
+    @mock.patch('avocado.utils.process.get_children_pids')
+    @mock.patch('avocado.utils.process.time.time')
+    @mock.patch('avocado.utils.process.time.sleep')
+    @mock.patch('avocado.utils.process.pid_exists')
+    def test_kill_process_tree_wait3s(self, pid_exists, sleep, p_time,
```

Comment:
Minor suggestion: naming this `test_kill_process_tree_wait_3s` slightly improves the readability IMO.

Reply:
sure